### PR TITLE
Add CDI client getter

### PR DESCRIPTION
**Pull request by awels**
- Added code to get an instance of a CDI client, copied from PR #328

Signed-off-by: Alexander Wels <awels@redhat.com>

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -76,6 +78,17 @@ func PanicOnError(err error) {
 	}
 }
 
+// Gets an instance of a kubernetes client that includes all the CDI extensions.
+func GetCDIClient() *clientset.Clientset {
+
+	cfg, err := clientcmd.BuildConfigFromFlags(master, kubeconfig)
+	PanicOnError(err)
```
Comment (jeffvance) [inline tests/utils.go]: panic vs Fail?

File: tests/utils.go [hunk 2]
```diff
@@ -76,6 +78,17 @@ func PanicOnError(err error) {
...
+	cdiClient, err := clientset.NewForConfig(cfg)
+	PanicOnError(err)
```
Comment (jeffvance) [inline tests/utils.go]: ditto

**Comment by jeffvance** [comment]
In func GetKubeClientFromRESTConfig:
```
 func GetKubeClientFromRESTConfig(config *rest.Config) (*kubernetes.Clientset, error) {
 ...
 	coreClient, err := kubernetes.NewForConfig(config)
 	if err != nil {
 		return nil, err
 	}
 	return coreClient, nil
 }
```
why not just: `return  kubernetes.NewForConfig(config)`  ?

File: tests/utils.go (hunk 1 above)
Comment (awels) [reply tests/utils.go]: So the way I read panic vs Fail, is that panic is 100% fatal, while Fail is not good, other tests might still work.

Comment (awels) [reply tests/utils.go]: Not quite sure what GetCDIclientOrDir() means, the OrDir part is confusing me. Is it a typo and should be OrDie?

Comment (jeffvance) [reply tests/utils.go]: Sorry. Fat finger on my part. Yes I meant `GetCDIclientOrDie`

**Comment by jeffvance** [comment]
@awels what do you think of renaming _tests/util.go_ to _tests/framework/util.go|framework.go_?

**Comment by awels** [comment]
@jeffvance That is an interesting question, and my answer will be it depends. I want to avoid trying to prematurely organize files into directories where we end up with a bunch of directories with one or two files in it. On the other hand we don't want to end up with a ton of files in the tests directory, where it is hard to find what you are looking for.

At this point we don't have many files yet, so I think I would prefer to keep everything in tests for now.

**Comment by awels** [comment]
So anything stopping this one from getting merged?

**Comment by jeffvance** [comment]
lgtm

**Pull request merged**